src/Tree/HydroTree.cpp

```diff
@@ -1153,7 +897,7 @@ void HydroTree<ndim,ParticleType,TreeCell>::UpdateHydroExportList
           {
             index_cell = Ncellexport[j]++;
           }
-          cellexportlist[j][index_cell] = cellptr;
+          cellexportlist[j][index_cell] = cell.id ;
```

Comment:
It just the position in the array. At the moment it is useful, so that is perhaps a good reason to store it. We could change how we work with cells to avoid the problem though.